pkg/storagepoolclaim/v1alpha1/storagepoolclaim.go

```diff
@@ -24,6 +24,12 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+// SupportedDiskType is a map containing the valid disk type
+var SupportedDiskType = map[string]bool{
```

Comment:
What is the need to use string as key? Can we use the apis type as the key?

Reply:
Updated